documentation/library_guide/parallel_api/pass_data_algorithms.rst

```diff
@@ -233,5 +267,94 @@ data transformation pipelines that also can be used with parallel range algorith
     return 0;
   }
 
+.. _use-iterators:
+
+Use |onedpl_short| Iterators
+----------------------------
+
+|onedpl_short| provides a set of `iterators <iterators-details>`_ that can be used to pass data to algorithms, in
```

Comment:
```suggestion
|onedpl_short| provides a set of `iterators <iterator-details>`_ that can be used to pass data to algorithms, in
```

Reply:
It seems that 
`indirectly device accessible <indirectly-device-accessible>`_ and `this table <iterator-properties-table>`_ links also do not work properly.

Reply:
Thanks, I think this is now fixed with your accepted changes.